requirements_dev.txt

```diff
@@ -8,6 +8,7 @@ mypy-protobuf
 pre-commit
 pytest
 pytest-asyncio
+python-dotenv
```

Comment:
why is this needed?

Reply:
We already use https://docs.pydantic.dev/latest/concepts/pydantic_settings/#dotenv-env-support, I'd rather we didn't introduce a dependency for this